packages/pwa/app/pwa-kit.config.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import path from 'path'
+import Config from 'pwa-kit-react-sdk/config'
+
+// The Config class automatically validate the config object.
+// Error will be thrown if configuration object is invalid.
+export default new Config({
+    version: '1.0.0',
+    app: {
+        commerceApi: {
+            clientId: 'c9c45bfd-0ed3-4aa2-9971-40f88962b836',
+            organizationId: 'f_ecom_zzrf_001',
+            shortCode: '8o7m175y',
+            siteId: 'RefArchGlobal'
+        },
+        einsteinApi: {
+            clientId: '1ea06c6e-c936-4324-bcf0-fada93f83bb1',
+            siteId: 'aaij-MobileFirst'
+        }
+    },
+    server: {
+        // The build directory (an absolute path)
+        buildDir: path.resolve(process.cwd(), 'build'),
```

Comment:
The SDK takes an absolute path for `buildDir`, that's why i need to use `.js` file format in order to have dynamic content.